web3-utils/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "ganache",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "deploy": "ts-node utils/deploy.ts",
+    "migrate": "truffle migrate"
+  },
```

Comment:
Sitll not sure why do we have Ganache and Truffle here. Could you elaborate?

Reply:
As mentioned in a previous comment, I think this is a reminiscence of the original approach for this PR.  It can be removed.